File: 250225/1부터 특정 수까지의 합 2/sum-from-1-to-a-certain-number-2.cpp
#include <iostream>

using namespace std;

int n;

int get_total(int n)
{
    if (n == 1)
        return 1;
    return get_total(n - 1) + n;
}

int main() {
    cin >> n;

    cout << get_total(n);
    return 0;
}

File: 250225/1부터 특정 수까지의 합 2/sum-from-1-to-a-certain-number-2.java
import java.util.*;
import java.io.*;

public class Main {
    static BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
    static int n;

    public static int getTotal(int n) {
        if (n == 1)
            return 1;
        return getTotal(n - 1) + n;
    }

    public static void main(String[] args) throws IOException {
        n = Integer.parseInt(br.readLine());

        System.out.println(getTotal(n));
    }
}